openfisca_uk/parameters/hmrc/tax_credits/child_tax_credit/takeup.yaml

```
@@ -0,0 +1,22 @@
+description: Share of eligible Child Tax Credit recipients that participate
```

```suggestion
description: Share of eligible Child Tax Credit families that participate
```

This paragraph from [page 4](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/851148/Child_Benefit__Child_Tax_Credit_and_Working_Tax_Credit_take-up_rates_2016_to_2017_restated.pdf#page=4) is poorly written but I think this is an accurate clarification:
>Child Benefit take-up rates measure the proportion of eligible children and young people who have Child Benefit claimed on their behalf. Tax credit take-up rates measure the proportion of eligible families who claim (the caseload take-up rate), as well as the proportion of available financial support which is claimed (the expenditure take-up rate)1